docs/upgrade/v1-1-2-to-v1-2-0.md

```diff
@@ -337,6 +339,78 @@ If you notice the upgrade is stuck in the **Upgrading System Service** state for
 - Related issue
   - [[BUG] The rancher-monitoring Pod stuck at terminating status when upgrading from v1.1.2 to v1.2.0-rc6](https://github.com/harvester/harvester/issues/4484)
 
+#### Many PODs in cattle-monitoring-system namespace are to be deleted
+
+1. Check the log of the `apply-manifests` pod to see if the following messages repeat.
+
+    ```
+    there are still 10 pods in cattle-monitoring-system to be deleted
+    Fri Dec  8 19:06:56 UTC 2023
+    there are still 10 pods in cattle-monitoring-system to be deleted
+    Fri Dec  8 19:07:01 UTC 2023
+    ```
+    When it continues to show 10 (or other number) pods, it encounters below issue.
+
+1. The root cause:
```

Comment:
```suggestion

    When the repeated messages show the same number of pods, the following issue has occurred.
```

Reply:
Displaying the root cause does not involve any user action so it should not be presented as a step in the procedure.